modules/odo-core-concepts.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/developer_cli_odo/understanding-odo.adoc
+
+:_content-type: CONCEPT
+[id="odo-core-concepts_{context}"]
+
+= odo core concepts
+
+`{odo-title}` abstracts Kubernetes concepts into a developer friendly terminology:
```

Comment:
```suggestion
`{odo-title}` abstracts Kubernetes concepts into a developer-friendly terminology:
```

Reply:
How about "Red Hat® OpenShift® Do ( odo ) is a command line interface (CLI) tool for creating applications on OpenShift Container Platform and Kubernetes." See https://www.ibm.com/docs/SSCSJL_4.3.x/techpreview/overview-odo.html#:~:text=Red%20Hat%C2%AE%20OpenShift%C2%AE,complexity%20of%20the%20underlying%20technology.

Reply:
How about inserting this up front: "Red Hat® OpenShift® Do ( odo ) is a command line interface (CLI) tool for creating applications on OpenShift Container Platform and Kubernetes." Borrowed from [IBM's description of odo](https://www.ibm.com/docs/SSCSJL_4.3.x/techpreview/overview-odo.html#:~:text=Red%20Hat%C2%AE%20OpenShift%C2%AE,complexity%20of%20the%20underlying%20technology.).

Reply:
I removed `-friendly` in response to the "Claims and recommendations" guideline, "Do not make unsubstantiated claims about the following items:
- Perceived benefits or characteristics of a product: Avoid phrases such as "simple to use,"
"effortless," "user-friendly," "future-proof," and "easy to understand." Users might not find these
statements true, and inflated language can reduce the credibility of the information."

Reply:
> How about this ^^^ (I borrowed that first sentence from [IBM's description of odo](https://www.ibm.com/docs/SSCSJL_4.3.x/techpreview/overview-odo.html#:~:text=Red%20Hat%C2%AE%20OpenShift%C2%AE,complexity%20of%20the%20underlying%20technology.).)

That IBM content looks ridiculous to me - calling it "Red Hat OpenShift Do" , is like using "Red Hat OpenShift C" for the oc client. Would anyone recognize what "Red Hat OpenShift Do" is? (the upstream github  only makes it to page 3 in google search)?

I guess the official name is "Red Hat OpenShift Developer CLI (odo)"

Reply:
> It refers to the following Kubernetes concepts using software development terminology.

I'm not sure this is accurate. My understanding is that the terminology is odo-specific.

The  glossary maps the odo terminology to  Kubernetes/development concepts